cmd/policy_ls_groups.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/pulumi/pulumi/pkg/apitype"
+	"github.com/pulumi/pulumi/pkg/backend/display"
+	"github.com/pulumi/pulumi/pkg/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyGroupsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "groups",
```

Comment:
I think we want to make this singular rather than plural, to be consistent with the rest of the CLI commands, e.g.

```
pulumi stack ls
pulumi stack tag ls
pulumi plugin ls

...

pulumi policy group ls
```